### DISCUSSION TITLE
Does the five fold CV is necessary for precision?

### DISCUSSION THREAD

**Discussion by mchlsdrv** [MIC-DKFZ/nnUNet#520] [Q&A] [answered]
Hi, if I'm interested only in the 3D fullres setting, is it necessary  to run all the 5-folds? I mean, will the model predict better after I've ran all the 5-folds vs. if I ran only the first fold? If yes, what is the file that keeps the weights etc. in it?

Thanks in advance,
Michael

**Comment by FabianIsensee** [comment, marked as answer]
Hi,
please read the common questions documentation. It's in there :-)

https://github.com/MIC-DKFZ/nnUNet/blob/master/documentation/common_questions.md#do-i-need-to-always-run-all-u-net-configurations

tldr:

```bash
nnUNet_train 3d_fullres nnUNetTrainerV2 TASKID all
nnUNet_predict -i XXX -o YYY -t TASKID -f all -m 3d_fullres
```

Best,
Fabian

**Reply by FabianIsensee** [reply]
Performance will be better of course when you train the five folds and use them as an ensemble. I cannot tell you by how much though

**Reply by mchlsdrv** [reply]
`-f all` setting means that you use an ensamble of the folds?

**Reply by FabianIsensee** [reply]
No this will train one model on all training cases
